OG/OG/AST/AntlrToProgramAST.cs

```diff
@@ -0,0 +1,93 @@
+using System;
+using System.Collections.Generic;
+using OG.AST.MachineSettings;
+namespace OG.gen
+
+{
+    public class AntlrToProgramAST:OGBaseVisitor<string>, ISemanticErrorable
```

Comment:
Jeg tænker det vi ønsker at få ud, vel er en \<ASTNode> eller \<ProgramNode> istedet for  \<string>?

Reply:
Enig